src/Compilers/CSharp/Test/Emit2/FlowAnalysis/FlowTests.cs

```diff
@@ -5558,5 +5558,83 @@ public static void Test()
                 Diagnostic(ErrorCode.ERR_UseDefViolation, "obj").WithArguments("obj").WithLocation(28, 17)
                 );
         }
+
+        [Fact]
+        [WorkItem(60645, "https://github.com/dotnet/roslyn/issues/60645")]
+        public void TestLocalConstantIsUsedInLambdaAttribute()
```

Comment:
It might be good to somehow demonstrate that used local constants lack a diagnostic, but unused local constants do have a diagnostic.